### PR TITLE
Fix: Token amount max and min

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description

- Shows error when negative value or value greater than max token amount is given when setting up ERC20 Voting DAO

Task: [APP-501](https://aragonassociation.atlassian.net/browse/APP-501)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [x] I have tested my code on the test network.

### CODE REVIEW

File: packages/web-app/src/components/addWallets/row.tsx [hunk 1]
```diff
@@ -32,6 +33,12 @@ export type WalletField = {
   amount: string;
 };
 
+// largest decimal that can be represented in 224 bits
+// before adding the 18 decimals
+const MAX_TOKEN_AMOUNT = BigNumber.from(
```
Comment (WissenIstNacht) [inline packages/web-app/src/components/addWallets/row.tsx]: Isn't this constant given by Ethers or something? Also, could you move to the constants?

**Review by WissenIstNacht — APPROVED** [review]
Thanks for the fix Fabrice!

File: packages/web-app/src/components/addWallets/row.tsx (hunk 1 above)
Comment (Fabricevladimir) [reply packages/web-app/src/components/addWallets/row.tsx]: Unfortunately it isn't no

**Review by sepehr2github — APPROVED** [review]
Thanks, Fabrice! It seems Good to me.

**Pull request merged**